jenkins/ci.suse.de/openstack-ardana.yaml

```diff
@@ -332,6 +338,12 @@
                   update.yml
           fi
 
+          # Deploy CaaSP
+          if [ -n "$want_caasp" ]; then
```

Comment:
you're defaulting to "0", but checking if the string is "nonempty" which means it will always deploy. There is some inconsistency here-- I suggest to do a multiple choice parameter than  a string